<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>
    <meta name="author" content="Yaqiang Cao" />
    <meta name="keyworkds" content="Yaqiang Cao, Homepage" />
    <meta name="description" content="Homepage of Yaqiang Cao" />
    <meta name="keywords" content="Yaqiang Cao" />
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <link rel="stylesheet" href="style.css" type="text/css" />
    <title>Yaqiang Cao Research CV</title>
</head>

<body data-feedly-mini="yes">
    <div id="layout-content">

        <script type="text/javascript">
            < !--
                function toggleBibtex(articleid) {
                    var bib = document.getElementById(articleid);
                    if (bib.style.display == "none") {
                        bib.style.display = "";
                    }
                    else {
                        bib.style.display = "none";
                    }
                }
            -->

        </script>

        <script type="text/javascript">
            var _gaq = _gaq || [];
            _gaq.push(['_setAccount', 'UA-40926388-1']);
            _gaq.push(['_trackPageview']);
            (function () {
                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
            })();

        </script>

        <table class="imgtable">
            <tr>
                <div id="toptitle">
                    <h1 style="text-align:center">
                        <a href="http://yaqiangcao.github.io/">Yaqiang Cao</a> &nbsp;(曹亚强), Ph.D.
                    </h1>
                </div>

                <p style="text-align:center">
                    Computational Biology Postdoc at <a href="https://www.nhlbi.nih.gov">NHLBI/NIH</a>, Bethesda,
                    Maryland, US.
                    <br />
                    <br>
                    Email: <a href="mailto:caoyaqiang0410@gmail.com">caoyaqiang0410@gmail.com</a> or <a
                        href="mailto:yaqiang.cao@nih.gov">yaqiang.cao@nih.gov</a><br />
                </p>

                <p style="text-align:center">
                    <a href="./YaqiangCao_CV_2022.pdf" data-toggle="tooltip" title="CV" target="_blank">
                        <img src="icons/resume.png" height="45px">
                    </a>
                    <a href="https://github.com/YaqiangCao" data-toggle="tooltip" title="GitHub" target="_blank">
                        <img src="icons/github.png" height="45px">
                    </a>
                    <a href="https://orcid.org/0000-0002-4665-1517" data-toggle="tooltip" title="ORCiD" target="_blank">
                        <img src="icons/orcid.png" height="45px">
                    </a>
                    <a href="https://scholar.google.com/citations?user=fr0xqysAAAAJ&hl=en" data-toggle="tooltip"
                        title="Google Scholar" target="_blank">
                        <img src="icons/google_scholar.png" height="45px">
                    </a>
                    <a href="https://www.researchgate.net/profile/Cao-Yaqiang" data-toggle="tooltip"
                        title="Research Gate" target="_blank">
                        <img src="icons/researchgate.png" height="45px">
                        <a href="https://publons.com/researcher/4753260/yaqiang-cao/" data-toggle="tooltip"
                            title="Publons" target="_blank">
                            <img src="icons/publons.png" height="45px">
                        </a>
                        <a href="https://loop.frontiersin.org/people/1416456/overview" data-toggle="tooltip"
                            title="Frontiers Loop" target="_blank">
                            <img src="icons/loop.png" height="45px">
                        </a>
                        <a href="https://twitter.com/shuye0410" data-toggle="tooltip"
                            title="Twitter" target="_blank">
                            <img src="icons/twitter.png" height="45px">
                        </a>
                </p>
            </tr>
        </table>


        <h2>Brief Research Biograph</h2>
        <div>
            <p> Yaqiang was enrolled in the major of bioengineering for college, before which he nearly knew nothing
                about the computer. He lost interest in making fermenters during his second/third year in college and
                found mathematical modeling and coding were quite full of fun when he participated in some competitions
                such as <a href="http://en.mcm.edu.cn">CUMCM</a>. Then he found the field of
                bioinformatics/computational biology.
            </p>
            <p>
                Yaqiang got his Ph.D. in Computational Biology from <a href="https://www.picb.ac.cn/picben">CAS-MPG
                    Partner Institute for Computational
                    Biology</a>, <a href="http://english.shb.cas.cn/Institutes/sifbs/">Shanghai Institutes for
                    Biological Sciences</a>, <a href="https://english.ucas.ac.cn">University of Chinese Academy of
                    Sciences</a> under the supervision of Dr. Jing-Dong
                Jackie Han. He is now a bioinformatics Postdoc in NHLBI/NIH, under the mentoring of Dr. Keji
                Zhao. </p>
            <p>Yaqiang has more than ten years of experience applying and analyzing deep sequencing technologies
                (RNA-Seq, ChIP-Seq, DNase-Seq, ATAC-seq, Hi-C, etc.) to understand transcription regulation
                during multiple biological processes of molecular systems. He focused on the epigenetic
                roles of human transposable elements during his Ph.D. student studies. Yaqiang is now working on
                understanding the chromatin interaction landscapes of human and mouse immune cells, especially
                on T cells. Collaborating closely with wet-lab mates, he analyzes and integrates sequencing data
                to intercept biology insights, and develops the analysis methods for new chromatin 1D and 3D
                profiling sequencing protocols.</p>
            <p>Yaqiang’s initial research interests include data mining, machine learning from massive
                sequencing data, and bioinformatics tools development rooted in Dr. Han’s lab. His interests
                turned to sequencing-based methodologies development and T-cells immunology during the Postdoc
                training with Dr. Zhao.</p>
            <p>Yaqiang has published the first/co-first author’s research articles on <b>Nucleic Acids Research
                    (2022 and 2020)</b>, <b>Blood (2020)</b>, <b>Bioinformatics (2020)</b>, and <b>Genome
                    Research (2019)</b>; many other solid interesting research papers as contributing
                collaborator such as <b>Science (2022)</b>, <b>Immunity (2022a&b)</b>, <b>Nature Cell Biology (2022)</b>, <b>Nature Metabolism (2020)</b>, <b>Science
                    Advances (2020)</b>, <b>Cell Reports (2019)</b>, <b>Cell Research (2015)</b> and so on.</p>
            <p>Yaqiang is also an active reviewer for Frontiers in Immunology, Frontiers in Oncology, Frontiers in Pharmacology, Frontiers In Genetics and
                BMC
                Genomics, happy to serve the community.</p>
            <p>Outside the working time, Yaqiang is interested in the research topics he is not involved in yet
                (not limited to biology). Diversity of knowledge is fantastic.</p>
        </div>


        <h2>
            Experience and Education Background
        </h2>
        <ul>
            <li>
                <p>
                    [2019.04-] National Heart, Lung, and Blood Institute (NHLBI), NIH, Postdoc, with <a
                        href="https://scholar.google.com/citations?user=kVmVQG8AAAAJ&hl=en">Dr. Keji Zhao</a>,
                    Maryland, US.
                </p>
            </li>
            <li>
                <p>
                    [2012.09 - 2018.12] CAS-MPG Partner Insitute for Computational Biology, Ph.D. of Computational
                    Biology, with <a href="http://www.cls.edu.cn/english/PrincipalInvestigator/pi/index4690.shtml">Dr.
                        Jing-Dong Jackie Han</a>, Shanghai, China.
                </p>
            </li>
            <li>
                <p>
                    [2008.09 - 2012.06] East China University of Science and Technology, Bachelor of Bioengineering,
                    Shanghai, China.
                </p>
            </li>
        </ul>

        <h2>
            Research Interests
        </h2>
        <ul>
            <p>gene expression regulation, epigenetics, bioinformatics, next-generation sequencing, chromatin 3D
                structures, T-cell immunology</p>
        </ul>


        <h2>Research Publications (<a href="https://scholar.google.com/citations?user=fr0xqysAAAAJ&hl=en">full
                list</a>)</h2>

        <h3>bioRxiv first or co-first preprint</h3>
        <ol>
            <li>
                Cao, Yaqiang, et al. “Hi-TrAC reveals fractal nesting of super-enhancers” bioRxiv (2022).
                <a href="https://www.biorxiv.org/content/10.1101/2022.07.13.499926v1.full">LINK</a>
            </li>
            <li>
                Liu, Shuai, et al. “Hi-TrAC reveals fine-scale chromatin structures organized by transcription factors”
                bioRxiv (2022). <a href="https://www.biorxiv.org/content/10.1101/2022.06.01.494329v1.full">LINK</a>
            </li>
        </ol>


        <h3>First/co-first Author Papers</h3>
        <h4>Bioinformatics Algorithms/Tools</h4>
        <ol>
            <li>
                Cao, Yaqiang, et al. "cLoops2: a full-stack comprehensive analytical tool for chromatin interactions."
                <i><b>Nucleic acids research</b></i> 50.1 (2022): 57-71.
                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab1233/6470683">LINK</a>
            </li>
            <li>
                Cao, Yaqiang, et al. “Accurate loop calling for 3D genomic data with cLoops.”
                <i><b>Bioinformatics</b></i> 36.3
                (2020): 666-675.
                <a href="https://academic.oup.com/bioinformatics/article/36/3/666/5553098">LINK</a>
            </li>
        </ol>

        <h4>Biological Questions Orientated Data Mining</h4>
        <ol>
            <li>
                Zeng, Yingying, et al. “Characterization of functional transposable element enhancers in acute myeloid
                leukemia.” <i><b>Science China Life Sciences</b></i> 63.5 (2020): 675-687.
                <a
                    href="http://engine.scichina.com/publisher/scp/journal/SCLS/doi/10.1007/s11427-019-1574-x?slug=abstract">LINK</a>
            </li>
            <li>
                Cao, Yaqiang, et al. “Widespread roles of enhancer-like transposable elements in cell identity and
                long-range genomic interactions.” <i><b>Genome research</b></i> 29.1 (2019): 40-52.
                <a href="https://genome.cshlp.org/content/29/1/40.short">LINK</a>
            </li>
        </ol>

        <h4>Sequencing Data Analysis Collaboration</h4>
        <ol>
            <li>
                Han, Mengmeng, et al. "A role for LSH in facilitating DNA methylation by DNMT1 through enhancing UHRF1
                chromatin association." <i><b>Nucleic acids research</b></i> 48.21 (2020): 12116-12134.
                <a href="https://academic.oup.com/nar/article/48/21/12116/5973463">LINK</a>
            </li>

            <li>
                Zhen, Tao, et al. "RUNX1 and CBFβ-SMMHC transactivate target genes together in abnormal myeloid
                progenitors for leukemia development." <i><b>Blood</b></i> 136.21 (2020): 2373-2385.
                <a
                    href="https://ashpublications.org/blood/article/doi/10.1182/blood.2020007747/463758/RUNX1-and-CBF-SMMHC-transactivate-target-genes?casa_token=z4O0k0BGXXIAAAAA:jbnSrdUS0oKWcr2zUecv8RKqbdlpGIk34e9DY9zFl37vI6ju5Nenifie2Vq_0ohBfDIv7XFY">LINK</a>
            </li>

            <li>
                Huang, Fei, et al. “BMP2 signalling activation enhances bone metastases of non‐small cell lung cancer.”
                <i><b>J Cell Mol Med</b></i>. 2020; 00: 1– 17.
                <a href="https://doi.org/10.1111/jcmm.15702">LINK</a>
            </li>

            <li>
                Wang, Lu, et al. “Activin/Smad2-induced histone H3 Lys-27 trimethylation (H3K27me3) reduction is crucial
                to initiate mesendoderm differentiation of human embryonic stem cells.” <i><b>Journal of Biological
                        Chemistry</b></i> 292.4 (2017): 1339-1350.
                <a href="http://www.jbc.org/content/292/4/1339.short">LINK</a>
            </li>
        </ol>

        <td align="bottom" colspan="1" style="margin-top:5px;">
            <h3>Involved Publications</h3>
            <div style="height: 400px; overflow: auto;">
                <ol>
                   <li>
                    Zhang, Xiaoli, et al. "KCNQ1OT1 promotes genome-wide transposon repression by guiding RNA–DNA triplexes and HP1 binding." <i><b>Nature Cell Biology</b></i> (2022):1-13.
                    </li>
 
                    <li>
                    Lin, Xueqiu, et al. "Nested epistasis enhancer networks for robust genome regulation." <i><b>Science</b></i> (2022): eabk3512.
                    </li>
                    <li>
                        Ren, Gang, et al. "Transcription factors TCF-1 and GATA3 are key factors for the epigenetic
                        priming of early innate lymphoid progenitors toward distinct cell fates." <i><b>Immunity</b></i>
                        55.8 (2022): 1402-1413.
                    <li>
                        Pan, Lixia, et al. “scPCOR-seq enables co-profiling of chromatin occupancy and RNAs in single
                        cells.”
                        <i><b>Communications Biology</b></i> 5.1 (2022): 1-9.
                    </li>
                    <li>
                        Fang, Difeng, et al. “Differential regulation of transcription factor T-bet induction during NK
                        cell
                        development and T helper-1 cell differentiation.” <i><b>Immunity</b></i> (2022):
                        S1074-7613(22)00128-5.
                    </li>
                    <li>
                        Huang, Fei, et al. “PNMA5 promotes bone metastasis of non-small-cell lung cancer as a target of
                        BMP2
                        signaling.” <i><b>Frontiers in cell and developmental biology</b></i> 9 (2021).
                    </li>
                    <li>
                        Ku, Wai Lim, et al. “Profiling single-cell histone modifications using indexing chromatin
                        immunocleavage
                        sequencing.” <i><b>Genome Research</b></i> (2021): gr-260893.
                    </li>
                    <li>
                        Xia, Xian, et al. “Three-dimensional facial-image analysis to predict heterogeneity of the human
                        ageing
                        rate and the impact of lifestyle.” <i><b>Nature Metabolism</b></i> (2020): 1-12.
                    </li>
                    <li>
                        Qiu, Xiang, et al. "Altered 3D chromatin structure permits inversional recombination at the IgH
                        locus."
                        <i><b>Science advances</b></i> 6.33 (2020): eaaz8850.
                    </li>
                    <li>
                        Chen, Xingwei, et al. “Immune cell types and secreted factors contributing to
                        inflammation-to-cancer
                        transition and immune therapy response.” <i><b>Cell reports</b></i> 26.7 (2019): 1965-1977.
                    </li>
                    <li>
                        Xu, Chi, et al. “Accurate Drug Repositioning through Non-tissue-Specific Core Signatures from
                        Cancer
                        Transcriptomes.” <i><b>Cell reports</b></i> 29.4 (2019): 1055.
                    </li>
                    <li>
                        Huang, Yi, et al. “Single-cell-level spatial gene expression in the embryonic neural
                        differentiation
                        niche.” <i><b>Genome research</b></i> 25.4 (2015): 570-581.
                    </li>
                    <li>
                        Chen, Weiyang, et al. “Three-dimensional human facial morphologies as robust aging markers.”
                        <i><b>Cell
                                research</b></i> 25.5 (2015): 574-587.
                    </li>
                </ol>
            </div>
        </td>

        <h2>Reviewer Services</h2>
        <ul>
            <li>
                <p><a href="https://www.frontiersin.org/journals/immunology">Frontiers in Immunology</a></p>
            </li>
            <li>
                <p><a href="https://www.frontiersin.org/journals/oncology">Frontiers in Oncology</a></p>
            </li>
            <li>
                <p><a href="https://www.frontiersin.org/journals/pharmacology">Frontiers in Pharmacology</a></p>
            </li>
            <li>
                <p><a href="https://www.frontiersin.org/journals/genetics">Frontiers In Genetics</a></p>
            </li>
            <li>
                <p><a href="https://bmcgenomics.biomedcentral.com/">BMC Genomics</a></p>
            </li>
            <li>
                <p><a href="https://journals.sagepub.com/home/ccx">Cancer Control</a></p>
            </li>
        </ul>

        <h2>GitHub Softwares</h2>
        <ul>
            <li>
                <p><a href="https://github.com/YaqiangCao/cLoops2">cLoops2</a>: full stack analysis tool for chromatin
                    interaction data such as Hi-TrAC (released in 2021)</p>
            </li>

            <li>
                <p><a href="https://github.com/YaqiangCao/cLoops">cLoops</a>: accurate and flexible loops calling tool
                    for 3D genomic data (released in 2017)</p>
            </li>

        </ul>


        <!-- BEGIN: Powered by Supercounters.com -->
        <center>
            <script type="text/javascript" src="//widget.supercounters.com/ssl/flag.js"></script>
            <script type="text/javascript">sc_flag(1642297, "FFFFFF", "000000", "cccccc", 2, 8, 0, 0)</script>
            <br><noscript><a href="http://www.supercounters.com/">Flag Counter</a></noscript>
        </center>
        <!-- END: Powered by Supercounters.com -->



        <div id="footer">
            <div id="footer-text">
                </br>Last updated at 2022-07-26 by Yaqiang Cao.
            </div>
        </div>
    </div>


</body>

</html>
